config/packages/security.yaml

```diff
@@ -74,13 +70,6 @@ security:
                 authenticators:
                     - lexik_jwt_authentication.jwt_token_authenticator
 
-        api:
-            pattern: "%sylius.security.api_regex%/.*"
```

Comment:
Is this parameter defined in AdminApi bundle?

Reply:
Yes, it is - there's also no need for the firewall if the Admin API Bundle is not enabled.

Reply:
I just wanted to ensure, that we are not leaving its definition in the core 👍